pkgs/development/web/nodejs/v24.nix

```diff
@@ -47,7 +47,6 @@ buildNodejs {
     ++ [
       ./configure-armv6-vfpv2.patch
       ./disable-darwin-v8-system-instrumentation-node19.patch
-      ./bypass-darwin-xcrun-node16.patch
```

Comment:
I'm trying a cleaner approach in https://github.com/NixOS/nixpkgs/pull/434742, I'll update it once this one has landed

Reply:
Added a comment, and added back the patch so we don't break darwin sandbox builds until the other PR reaches unstable